plugins/janus_sip.c

```diff
@@ -2276,8 +2277,13 @@ void janus_sip_sofia_callback(nua_event_t event, int status, char const *phrase,
 			}
 			if(session->stack->s_nh_i != NULL) {
 				if(session->stack->s_nh_i == nh) {
-					/* re-INVITE, we don't support those. */
-					nua_respond(nh, 488, sip_status_phrase(488), TAG_END());
+					if (sdp_session_cmp(sdp_session(session->raw_media), sdp_session(parser)) == 0) {
```

Comment:
How does this method work? Does it only inspect media, ports and stuff like that, or everything in the SDP? Just wondering as my guess that even a tiny difference in the SDP may actually trigger a `!=0` there.